core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/RequesterPaysSetupServiceSpec.scala

```diff
@@ -99,23 +95,88 @@ class RequesterPaysSetupServiceSpec extends AnyFlatSpec with Matchers with Mocki
     runAndWait(workspaceRequesterPaysQuery.insertAllForUser(minimalTestData.workspace2.toWorkspaceName, userInfo.userEmail, Set(user1SA)))
     runAndWait(workspaceRequesterPaysQuery.insertAllForUser(minimalTestData.workspace.toWorkspaceName, minimalTestData.userReader.userEmail, Set(user2SA)))
     runAndWait(workspaceRequesterPaysQuery.insertAllForUser(minimalTestData.workspace2.toWorkspaceName, minimalTestData.userReader.userEmail, Set(user2SA)))
-    runAndWait(workspaceRequesterPaysQuery.userExistsInWorkspaceNamespace(minimalTestData.billingProject.projectName.value, userInfo.userEmail)) shouldBe true
-    runAndWait(workspaceRequesterPaysQuery.userExistsInWorkspaceNamespace(minimalTestData.billingProject.projectName.value, minimalTestData.userReader.userEmail)) shouldBe true
 
     // add 2 users to mock google bindings
     val initialBindings = Map(service.requesterPaysRole -> expectedEmails.map(expectedEmail => "serviceAccount:" + expectedEmail.client_email))
     service.googleServicesDAO.asInstanceOf[MockGoogleServicesDAO].policies.put(minimalTestData.workspace.googleProjectId, initialBindings)
 
     // remove users from 1 workspace and check that it did not get removed from google bindings
     service.revokeAllUsersFromWorkspace(minimalTestData.workspace).futureValue should contain theSameElementsAs expectedEmails
-    runAndWait(workspaceRequesterPaysQuery.userExistsInWorkspaceNamespace(minimalTestData.billingProject.projectName.value, userInfo.userEmail)) shouldBe true
-    runAndWait(workspaceRequesterPaysQuery.userExistsInWorkspaceNamespace(minimalTestData.billingProject.projectName.value, minimalTestData.userReader.userEmail)) shouldBe true
```

Comment:
I removed these checks because, to be honest, I think they are wrong. They are testing implementation details (that this function returns this specific information), rather than the actual functionality (that the policies are in google, or that we return users X or Y format and information)